Chapters/2-3-Chapter10.md

```diff
@@ -0,0 +1,295 @@
+
+"I believe I have a great deal to explain to you. This is my kaln, Linyu. We arrived here with you yesterday, but we used to live here many ma'to before."
+
+Linyu handed Kipnu a wooden bowl of some sort of fish stew, and he eagerly took it with gratitude and started eating.
+
+In between spoonfuls, he asked, "Why have you come back to a place like this?"
+
+"I spent three ma'to here at a dark time of my life. My journey began like most Ulukomai - a slave and a kanosin addict. The stuff was produced here by slave labor and then smugglers would bring it to the mainland. It's a vicious cycle that I got caught up in. Before I was an addict, I was a doctor - one of the best in Rudjan. I would often see kanosin addicts that had come directly from Ulukoma looking to escape their addiction and I developed my own treatment protocol that worked very well. In a stupidly brazen move, I decided to test it on myself to prove to my peers how effective it was, but I vastly underestimated its power and I was unable to pull out of it."
+
+"You were the cause of your own suffering."
+
+"Yes. My junds left me and I was sent here as punishment. This place used to be equitable. The clear-minded still held the power here, but they would try to uplift the addicted. Shinna changed all that, bastardized all of the traditions that kept this place livable for the addicted. Now death and addiction hang in this place."
+
+"I noticed... a class hierarchy back at the pavilion."
+
+"Shinna was a chemist in Shaki who somehow stumbled upon an antidote for Kanosin. Then thi came here with hae stockpile of antikan, and instead of giving it to us, thi used it as leverage to gain power over me. Now the upper class use it to get high at night and by morning they're back to normal."
+
+"Wait, gain power over you? So you were a leader here?"
+
+"Yes. One of three that formed a council of sorts. I was pushing for Ulukoma to become an independent province when I was ousted."
+
+"How did you go from an addict to the leader of Ulukoma?"
+
+"That's a long story," thi smiled and looked into hae mind, thinking of a better time.
+
+"I like long stories," he said. "They get me invested."
+
+Thi laughed, and paused to prepare haeself.
+
+"I guess I'll start where I left off. You remember I mentioned traditions. One of the traditions we used to have was the evening. It used to be a time of storytelling, singing, confessing of love, public mediation. Anyone could stand up and say or do anything they wanted. Then some would group up and explore their sexualities together before sleep, and others would just sleep if they weren't ready. The evening was a place and time where people weaved their lives together and apart and together again, and everyone grew closer to each other because of it. I had been under the addiction, but I had the faculty to understand their words and feel the love of a tribe more deeply than I knew of my junds."
```

Comment:
This paragraph feels like a mouth full and felt me confused as to what happened after it.